README.md

```diff
@@ -42,25 +42,53 @@ pip install matcha-ml
 matcha provision
 ```
 
-After the provisioning completes, run the following bash script
+After the provisioning completes, we will have on hand these resources:
+* Kubernetes cluster on Azure
+* Seldon Core installed on this cluster
+* Istio ingress installed on this cluster
+
+### Deploy model
+Apply the prepared kubernetes manifest to deploy the model:
 
 ```bash
-./setup_deploy.sh
+kubectl apply -f infrastructure/llm/seldondeployment.yaml
 ```
 
-Run the deployment pipeline.
+This will create a Seldon deployment, which consists of:
+* Pod, that loads the pipeline and the model for inference
+* Service and ingress routing, for accessing it from the outside
 
-```bash
-python run.py -d
+### Query model
+You can get ingress IP with matcha:
+```
+matcha get model-deployer base-url
 ```
 
-You can take a look [here](docs/data-version-control.md) for guidance on interacting with data version control on this project.
+Full URL to query the model:
+```
+http://<INGRESS_IP>/seldon/matcha-seldon-workloads/llm/v2/models/transformer/infer
+```
+
+The expected payload structure is as follows:
+```json
+{
+    "inputs": [
+        {
+            "name": "array_inputs",
+            "shape": [-1],
+            "datatype": "string",
+            "data": "String to summarize"
```

Comment:
I think `data` payload should be changed as it's a text-to-text generation task.

Reply:
It's also worth mentioning the format expected by flan-t5 model to be a question-answering task.

Reply:
:+1: